simpeg_drivers/uijson.py

```diff
@@ -0,0 +1,71 @@
+# '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2025 Mira Geoscience Ltd.                                          '
+#                                                                                   '
+#  This file is part of simpeg-drivers package.                                     '
+#                                                                                   '
+#  simpeg-drivers is distributed under the terms and conditions of the MIT License  '
+#  (see LICENSE file at the root of this source code package).                      '
+#                                                                                   '
+# '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+import json
+import logging
+from pathlib import Path
+from typing import Annotated
+
+from geoh5py.groups import SimPEGGroup, UIJsonGroup
+from geoh5py.shared.validators import empty_string_to_none, none_to_empty_string
+from geoh5py.ui_json.enforcers import UUIDEnforcer
+from geoh5py.ui_json.ui_json import BaseUIJson
+from pydantic import BeforeValidator, PlainSerializer, field_validator
+
+import simpeg_drivers
+
+
+logger = logging.getLogger(__name__)
+
+
+OptionalPath = Annotated[
+    Path | None,
+    BeforeValidator(empty_string_to_none),
+    PlainSerializer(none_to_empty_string),
+]
+
+
+class SimPEGDriversUIJson(BaseUIJson):
+    version: str = simpeg_drivers.__version__
+    title: str
+    icon: str
+    documentation: str
+    conda_environment: str
+    run_command: str
+    geoh5: Path | None
+    monitoring_directory: OptionalPath
+    workspace_geoh5: OptionalPath
```

Comment:
Aren't these already on the base class?

Reply:
Yep, I was just trying to set the order here, but probably not too important.. I've moved onto the base class